src/librbd/internal.h

```diff
@@ -41,6 +41,9 @@ enum {
   l_librbd_readahead,
   l_librbd_readahead_bytes,
 
+  l_librbd_flush_cache,
+  l_librbd_invalidate_cache,
```

Comment:
The ```l_librbd_flush_cache``` can just use the existing ```l_librbd_flush``` perf counter.  As for ```l_librbd_invalidate_cache```, to me it is in-line with the other function-level perf counters such as ```l_librbd_snap_xyz```, ```l_librbd_resize```, etc.